### ISSUE TITLE
VM CPU usage at 100% when using ie URL analysis package

### ISSUE THREAD

**Issue by seanthegeek** [ctxis/CAPE#333]
Some recent change in the analysis package is causing CPU usage to stay at 100% without IE actually opening during a URL analysis with the ie package.

**Comment by kevoreilly** [comment]
Hi Sean, sorry not to get back to you earlier - are you still seeing this?

**Comment by seanthegeek** [comment]
Yes, I'm still seeing this after pulling the latest commit this morning

**Comment by seanthegeek** [comment]
@kevoreilly Bump

**Comment by seanthegeek** [comment]
Also occurs on your hosted instance. https://cape.contextis.com/analysis/87836/

**Comment by kevoreilly** [comment]
Hmm I just tried with old loader and it seemed to work. So possibly an issue with the new loader and IE. Let me look into it.

**Comment by seanthegeek** [comment]
@kevoreilly Have you had a chance to look at this?

**Comment by kevoreilly** [comment]
Yes I just spent a while digging into this and have found the problem is due to a monitor code change which was attempting to mitigate problems with IcedID samples caused by a measure put in to allow VBCrypter samples to run (https://github.com/kevoreilly/capemon/commit/f4fe2d5470bf5fadd0b3f502caad1be83f34a977).

I've just compiled a monitor reverting just this change and IE seems to load up again - please give the attached monitor a go and let me know.

I'm not sure how to best fix this whilst keeping compatability with VBCrypter samples - I'll need to speak to the researcher who made that mod in the first place and work out if there isn't a better way.

[capemon.zip](https://github.com/ctxis/CAPE/files/3643956/capemon.zip)

**Comment by seanthegeek** [comment]
@kevoreilly My VMs are 64 bit. Can you build a 64 bit version?

**Comment by kevoreilly** [comment]
IE is 32-bit - even on 64-bit windoze ;-)

**Comment by seanthegeek** [comment]
Seeing the same issue. It goes in analyzer/windows/dll/, right?

**Comment by kevoreilly** [comment]
Oh dear - no IE window?

![image](https://user-images.githubusercontent.com/22219888/65462260-50259880-de4d-11e9-8f22-efece97269e3.png)

**Comment by kevoreilly** [comment]
I'm gonna have to call it a night as it's getting late over here - but progress has been made, am hopeful we will nail this bug once we get to the bottom of it.
